### PR TITLE
[DASH-2323] Add navigation bar in hosting files view

### PR DESCRIPTION
**Pull request by wpasinski**
I have to look at this after a day to see if I like the styles.

zrzut ekranu 2016-10-20 o 18 39 52

### CODE REVIEW

**Comment by wpasinski** [comment]
@gh3r I removed it at some point, but then I found it useful.

**Comment by gh3r** [comment]
@wpasinski Fine, let's leave it then

File: src/apps/Hosting/HostingFilesList.js [hunk 1]
```diff
@@ -200,9 +206,7 @@ const HostingFilesList = React.createClass({
   },
 
   renderDotsListItem() {
-    return (
-      <DotsListItem onClickDots={this.moveDirectoryUp} />
-    );
+    return <DotsListItem onClickDots={() => this.moveDirectoryUp()} />;
```
Comment (patrykkopycinski) [inline src/apps/Hosting/HostingFilesList.js]: `() => this.moveDirectoryUp()` => `this.moveDirectoryUp`

File: src/common/DirectoryNavigation/DirectoryNavigation.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+      display: 'flex',
+      alignItems: 'center',
+      cursor: 'pointer'
+    }
+  };
+
+  const handleFontIconClick = () => moveDirectoryUp(directoryDepth);
+
+  const renderLinks = () => {
+    const links = [];
```
Comment (patrykkopycinski) [inline src/common/DirectoryNavigation/DirectoryNavigation.js]: `.map()` fits better

File: src/common/DirectoryNavigation/DirectoryNavigationLink.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    fontIcon: {
+      fontSize: 18,
+      display: 'flex',
+      alignItems: 'center',
+      color: 'rgba(0, 0, 0, .4)'
+    }
+  };
+
+  const truncateLabel = (text) => {
+    const sideLettersCount = 8;
```
Comment (patrykkopycinski) [inline src/common/DirectoryNavigation/DirectoryNavigationLink.js]: move to props

**Pull request merged**